setup.sql

```diff
@@ -107,7 +109,8 @@ CREATE TABLE `CampaignNotePageDetail` (
   `updatedAt` datetime NOT NULL,
   `deletedAt` datetime DEFAULT NULL,
   PRIMARY KEY (`ID`),
-  KEY `detailInCampaign` (`Campaign_ID`,`PageDetail_ID`)
+  KEY `detailInCampaign` (`Campaign_ID`,`PageDetail_ID`),
+  KEY `notePartitionRange` (`partition_noteStart`,`partition_noteEnd`)
```

Comment:
Why do we need to keep track of this? I would have thought the db kept track on its own.

Reply:
This is an index.  So anytime we think we're going to use properties within a MySQL `WHERE` clause, we should probably have an index for it.  In this case, the API for the note would likely include a noteStart and noteEnd index and we'd use that to determine the note in the db to edit.